### ISSUE TITLE
Work show page needs admin sidebar

### ISSUE THREAD

**Issue by ggeisler** [projecthydra-labs/hyrax#814]
When a depositor is viewing their own work, such as after completing a deposit and navigating to a work from the Works table, the show page for the work should include the admin sidebar, but it currently does not:

![butterfly_research](https://cloud.githubusercontent.com/assets/101482/25286404/768b1918-2672-11e7-8352-3c5698701792.png)

**Comment by jcoyne** [comment]
This should probably be a new route then, right? So there is one route for "viewing this work in the context of my dashboard" and "viewing this work in public".  Presently we only have the latter.

**Comment by ggeisler** [comment]
Yes, I think that's right, separate routes for public viewing versus admin-context (depositing user or other user with privileges to the work) viewing.
